### DISCUSSION TITLE
[react/npm] setWalletFallback not working

### DISCUSSION THREAD

**Discussion by JerryZaz** [reach-sh/reach-lang#1101] [Q&A] [answered]
I made some changes to my project structure, now I'm launching with `npm start` instead of `reach react` but now, `reach.setWalletFallback(...)` is not working, I'm getting prompted to connect to MetaMask. 

The Reach library is initialized properly, I added a dummy `reach.formatCurrency` to double check, I have access to the sourcefiles and I'm able to click on the import to check that I have the right reference to ALGO_WalletConnect, but something is just not happening.

Dependencies on my package.json, not many things that could be causing a conflict.
```
"dependencies": {
    "@reach-sh/stdlib": "^0.1.10-rc.7",
    "react": "^18.1.0",
    "react-dom": "^18.1.0",
    "react-scripts": "^5.0.1"
  }
```

**Comment by jeapostrophe** [comment, marked as answer]
My assumption is that you need to call `loadStdlib` with an explicit `{ REACH_CONNECTOR_MODE: 'ALGO' }`
